### DISCUSSION TITLE
Cancel and stop all queries and mutations

### DISCUSSION THREAD

**Discussion by cubecleveland** [TanStack/query#3634] [Q&A]
Hello,
I have a situtation where we neeed to stop everything running including mutations, regardless of keys.
Can that thing be achieved ?

Basically clear everything in the system and not refetch. ( till user decides to do something else

**Comment by TkDodo** [comment]
you can clear everything with `queryClient.clear()`, which will also cancel and abort ongoing fetches if you use the provided `AbortSignal`. Running mutations cannot be stopped / cancelled.
